This error occurs when more that one `default` case exists in a switch statement.

```ts
let x = 123
switch (x) {
  case 1:
    console.log(1)
    break
  default:
    console.log('Unknown')
    break
  case 2:
    console.log(2)
    break
  default:
    throw new Error('Bad number')
    break
}
```

To fix the error, combine the `default` cases or remove the extraneous one.


```ts
let x = 123
switch (x) {
  case 1:
    console.log(1)
    break
  case 2:
    console.log(2)
    break
  default:
    console.log('Unknown')
    throw new Error('Bad number')
    break
}
```
